VMS/Pages/ActivityInfoPage/InfoPage.razor

```diff
@@ -0,0 +1,74 @@
+﻿<div class="InfoPage">
+    <div class="TopPage">
+        <span class="Return">
+            <a href="@Routes.ActivitySearch" class="Return-text">&#60;&#60; Quay lại</a>
```

Comment:
There are two "Quay lại" links in this page. Please remove one.

Reply:
done